extensions/wikia/ArticleVideoMVP/styles/premium-mvp.scss

```diff
@@ -0,0 +1,135 @@
+#premium-mvp-article-header {
```

Comment:
At Wikia we have a rule to target elements in JS by ID when possible and style using classes.
ID selectors are stronger and it makes it very hard to override them.